tests/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "opte-tests"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+zone = { git = "https://github.com/oxidecomputer/zone" }
+ztest = { git = "https://github.com/oxidecomputer/falcon", branch = "ztest" }
```

Comment:
Could we merge this branch and point this to `main` before merging?

Reply:
Done